python/dgl/graphbolt/base.py

```diff
@@ -185,6 +186,37 @@ def etype_str_to_tuple(c_etype):
     return ret
 
 
+def seed_type_str_to_ntypes(seed_type, seed_size):
+    """Convert seeds type to node types from string to list.
+
+    Examples
+    --------
+    1. node pairs
+
+    >>> seed_type = "user:like:item"
+    >>> seed_size = 2
+    >>> node_type = seed_type_str_to_ntypes(seed_type, seed_size)
+    >>> print(node_type)
+    ["user", "item"]
+
+    2. hyperlink
+
+    >>> seed_type = "user:item:user"
```

Comment:
user:item:user is not a very generic, maybe "query:user:item", but the "user:item:user" is a good corner example, just to clarify both user column will be uniqued together, right?

Reply:
> user:item:user is not a very generic, maybe "query:user:item", but the "user:item:user" is a good corner example, just to clarify both user column will be uniqued together, right?

Right. I will change the case to "query:user:item", and clarify that seeds with same ntype will be uniqued together in `_seed_preprocess` in subgraph sampler.